<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcp_testClass</class>
 <widget class="QWidget" name="tcp_testClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tcp_test</string>
  </property>
  <widget class="QLabel" name="m_StatusLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>等待连接：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>获取的数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送的数据</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_GetLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_SendLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_BtnSend">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tcp_test.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_BtnSend</sender>
   <signal>clicked()</signal>
   <receiver>tcp_testClass</receiver>
   <slot>SendMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SendMessage()</slot>
 </slots>
</ui>
